### ISSUE TITLE
allow safe strings from different languages than JS

### ISSUE THREAD

**Issue by tillsc** [complate/complate-stream#34]
When using other languages than JS:

It would be nice to pass "native" strings into complate marked as `safe`. complate should accept those Strings as HTMLStrings. 

We should discuss this.

**Comment by FND** [comment]
Makes sense. I guess that means we'd have to go full duck typing there, checking for a property rather than relying on inheritance (e.g. `!!node.htmlSafe` instead of `node instanceof HTMLString`)?
